api/pom.xml

```diff
@@ -18,6 +18,8 @@
             src/main/java/ca/bc/gov/educ/api/trax/config/**,
             src/main/java/ca/bc/gov/educ/api/trax/constant/**,
             src/main/java/ca/bc/gov/educ/api/trax/choreographer/**,
+            src/main/java/ca/bc/gov/educ/api/trax/messaging/**,
```

Comment:
I can understand removing scheduler from sonar, but why messaging?

Reply:
All other NATS feature enabled APIs get the messaging package put to sonar exclusion with the following reason.
Any changes made in it like a new topic is added or some NATS configuration is changed will cause the coverage issue on the new code even though the message package is in junit exclusion.